spring-boot-testing/src/main/java/com/baeldung/testlog/TestLog.java

```diff
@@ -0,0 +1,22 @@
+package com.baeldung.testlog;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class TestLog {
+
+	Logger LOG = LoggerFactory.getLogger(this.getClass());
```

Comment:
This is using tabs, rather than spaces for indentation. Please reformat using the Baeldung format guidelines.